apps/prairielearn/src/lib/code-caller/code-caller-native.ts

```diff
@@ -11,45 +11,47 @@ import { logger } from '@prairielearn/logger';
 import { deferredPromise } from '../deferred.js';
 import { APP_ROOT_PATH, REPOSITORY_ROOT_PATH } from '../paths.js';
 
-import { FunctionMissingError } from './code-caller-shared.js';
+import { type CodeCallerNativeChildProcess } from './code-caller-native-types.js';
```

Comment:
Nit: now that `code-caller-native` is itself in TypeScript, consider inlining this `CodeCallerNativeChildProcess` type in this file.